test/gc/Transforms/Pipeline/run.mlir

```diff
@@ -0,0 +1,23 @@
+// RUN: gc-opt %s --gc-cpu-pipeline | gc-cpu-runner -e main -entry-point-result=void | FileCheck %s
```

Comment:
We might need to add some complex example to cover the default pipeline.

Reply:
@LongshengDu Please help to add some test cases and check if the pipeline works for the current linalgx and onednngraph dialect. Maybe in another PR, or in this one?